packages/server/src/index.ts

```diff
@@ -1863,7 +1885,14 @@ export async function start(): Promise<void> {
 
     const io = new Server(server, {
         cors: {
-            origin: '*'
+            origin: function (origin: string | undefined, callback: (err: Error | null, allow?: boolean) => void) {
```

Comment:
the function looks the same as the one above, is it possible to refractor into a single function?

Reply:
Agreed. Addressed in 657dace.